src/actions/index.js

```diff
@@ -67,7 +67,7 @@ export const openFolder = (folderId, context = FILES_CONTEXT) => {
       context,
       folder: Object.assign(extractFileAttributes(folder), {
         parent: extractFileAttributes(parent)}),
-      files: folder.relations('contents').map(
+      files: (folder.relations('contents') || []).map(
```

Comment:
Yes, see [this](https://github.com/cozy/cozy-stack/blob/master/docs/files.md#response-1). I think `relationships.contents.data` used to be an empty array, and now it's null (when a folder has nothing in it, for example).

Not sure where this should be fixed, here or in the stack. I can live with both. Ping @aenario ?